decks.js

```diff
@@ -1,1194 +1,1293 @@
-          },
-          {
-              name: "Martian Manhunter",
-              content: {
-                  secondary: "👽🔥"
+              {
+                  name: "Sirius Black",
+                  content: {
+                      primary: "🐶",
+                      secondary: "◾🦅",
+                  },
+                  hint: "prisoner of azkaban",
+                  matchValue: "He was named godfather of Harry",
+                  seenHint: false,
               },
-              hint: "He can read your mind but is afraid of fire",
-              matchValue: `J'onn J'onzz`,
-              seenHint: false
-          },
-          {
-              name: "Wonder Woman",
-              content: {
-                  secondary: "👸⚔"
+              {
+                  name: "Albus Dumbledore",
+                  content: {
+                      primary: "💯",
+                      secondary: "🦅👨‍🏫",
+                  },
+                  hint: "100 points to gryffindor",
+                  matchValue: "Defeated Gellert Grindelwald in the global wizarding war",
+                  seenHint: false,
               },
-              hint: "Princess of the Amazons",
-              matchValue: `Diana Prince`,
-              seenHint: false
-          },
-          {
-              name: "Green Arrow",
-              content: {
-                  secondary: "🏹🟩"
+              {
+                  name: "Harry Potter",
+                  content: {
+                      primary: "⚡",
+                      secondary: "🦉🦅",
+                  },
+                  hint: "Chosen one",
+                  matchValue: "Becomes master of death in dealthly hallows",
+                  seenHint: false,
               },
-              hint: "He's another genius billionaire like Batman",
-              matchValue: `Oliver Queen`,
-              seenHint: false
-          },
-          {
-              name: "Constantine",
-              content: {
-                  secondary: "👻🗣"
+              {
+                  name: "Lord Voldemort",
+                  content: {
+                      primary: "7️⃣",
+                      secondary: "👃🐍",
+                  },
+                  hint: "You-Know-Who",
+                  matchValue: "Tom Marvolo Riddle",
+                  seenHint: false,
               },
-              hint: "Can talk to dead people",
-              matchValue: `John Constantine`,
-              seenHint: false
-          },
-          {
-              name: "Shazam",
-              content: {
-                  secondary: "🧙‍♂️🌩"
+          ]
+    },
+    {
+      name: "Cartoons",
+          author: "Ayushi Gautam",
+          instructions: "The emojis on each card represent your favourite childhood cartoons you loved!. You can flip(click) the card and see a hint.Drag and drop a card onto the matching text clue on the right, which is cartoon's name in easy mode and some info regarding cartoon in challengening mode.",
```

Comment:
"some info" is not very specific. Would it be possible to narrow it somehow? For example in the already existing decks you can see that challenging mode uses character names or places.

Reply:
yeah sure, I'll try to do that

Reply:
The punctuation is a bit messed up here (`!` followed by `.` and no space after `.`) so please fix that as well